### PR TITLE
Encourage R users to check out `animovement`

### LINKED ISSUE #305: Redirect R users to `animovement`
**Is your feature request related to a problem? Please describe.**
[animovement](https://www.roald-arboel.com/animovement/) is an R library that is very similar in scope and design.
The project's homepage acknowledges that and directs Python users our way.

> The primary aim of the animovement package is to provide a unified, standardised workflow for analysing animal movement data, in a tidyverse-friendly syntax. We work actively with the developers of the Python [movement](https://movement.neuroinformatics.dev/) package, to reach a similar data standards, workflow and use cases; if you prefer analysing your data in Python, we highly recommend using [movement](https://movement.neuroinformatics.dev/).

**Describe the solution you'd like**
We should have a similar text snippet in our homepages, directing R users towards `animovement`. Sth like: "if you are more fluent in R, than Python, use animovement".

### PR DESCRIPTION
**Pull request by niksirbi**
## Description

**What is this PR**

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Documentation

**Why is this PR needed?**

Closes #305 

**What does this PR do?**

Adds a "tip" admonition to the README and the website homepage. Sadly I had to duplicate the text in both places, because the syntax for creating admonition boxes is different in GitHub-flavored vs myst-markdown. We could do away with the duplication if we went for simpler rendering options, but I like how the admonitions look.

## How has this PR been tested?

Local docs build.

### CODE REVIEW

File: docs/source/snippets/animovement-tip.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:::{admonition} Tip for R users
```
Comment (lochhh) [inline docs/source/snippets/animovement-tip.md:1]: Use `:::{tip}` ?

Comment (niksirbi) [reply docs/source/snippets/animovement-tip.md:1]: If I do that, the custom "title"  I want to give is not recognised as a "title". 
So I've found this workaround with admonition, which allows me a custom header. Do you know of another way?

Comment (lochhh) [reply docs/source/snippets/animovement-tip.md:1]: Ah thanks, didn't know this!

**Review by sfmig — APPROVED** [review]
LGTM! nice shoutout ⭐ 

I was thinking about how we could avoid the duplication and was wondering if a script that converts between myst and github-flavoured markdown admonitions may be helpful. We already have two admonition duplicates, if we expect it to grow it may be worth it.

Maybe one way could be we programmatically generate some of the snippets .md files by searching for GFM admonitions in the README file. I had a quick go generating a converter script, but feel free to ditch if too complicated.

```python
import argparse
import re


def extract_gfm_admonitions_as_snippets(gfm_text):
    # Regex pattern to match GFM admonition structure
    matches = re.findall(r"^>\s?(.*)", gfm_text, re.MULTILINE)

    if not matches:
        return "No GitHub Markdown admonitions found."

    # Extract admonition type and content
    admonition_dict = {}
    for m_i, m in enumerate(matches):
        if m.startswith("[!"):
            admonition_dict[m.strip("[!]")] = matches[m_i + 1]

    # Convert GFM admonitions to MyST format
    for admonition_type, content in admonition_dict.items():
        # apply MyST formatting
        myst_admonition = (
            f"::{{admonition}} {admonition_type.capitalize()}\n"
            f":class: {admonition_type}\n"
            f"{content}\n:::"
        )

        # write to file
        with open(f"snippets/{admonition_type}.md", "w") as file:
            file.write(myst_admonition)


if __name__ == "__main__":
    parser = argparse.ArgumentParser()
    parser.add_argument("gfm_file", help="Path to GFM markdown file")

    args = parser.parse_args()

    # read a GFM markdown file
    with open(args.gfm_file) as file:
        gfm_text = file.read()

    # extract GFM admonitions as MyST-flavoured .md snippets
    myst_text = extract_gfm_admonitions_as_snippets(gfm_text)
    print(myst_text)
```

File: README.md [hunk 2]
```diff
@@ -34,10 +34,15 @@ We aim to support a range of pose estimation packages, along with 2D or 3D track
...
 
-## Status
 > [!Warning]
 > 🏗️ The package is currently in early development and the interface is subject to change. Feel free to play around and provide feedback.
 
+> [!Tip]
+> If you prefer analysing your data in R, we recommend checking out the
+> [animovement](https://www.roald-arboel.com/animovement/) toolbox, which is similar in scope.
+> We are working together with its developer
+> to gradually converge on common data standards and workflows.
```
Comment (sfmig) [inline README.md:44]: Just fyi, for the script I pasted to work as is the admonition needs to be in one line
```suggestion
> If you prefer analysing your data in R, we recommend checking out the [animovement](https://www.roald-arboel.com/animovement/) toolbox, which is similar in scope. We are working together with its developer to gradually converge on common data standards and workflows.
```

Comment (niksirbi) [reply README.md:46]: I modified the script such that it can correctly parse both cases, so the suggested modification is no longer necessary.

**Comment by niksirbi** [comment]
Hey @sfmig, thanks for sending me down a 🐰 🕳️  

I ended up writing such a script (using yours as a starting points). Not sure it's worth it, since for the sake of 2 admonitions we now have to maintain an entire scripts, bit I lean towards using the script anyway. My reasoning is that in the future we may build upon it to do some more intelligent parsing from `README.md` to `index.md`, in a way that further reduces duplication (not just across admonitions).

File: docs/make.bat [hunk 3]
```diff
@@ -28,14 +28,19 @@ if "%1" == "" goto help
...
 if "%1" == "clean" (
-	@echo Removing auto-generated files under 'docs' and 'src'...
+	echo Removing auto-generated files...
 	rmdir /S /Q %BUILDDIR%
 	del /Q %SOURCEDIR%\api_index.rst
     rmdir /S /Q %SOURCEDIR%\api\
 	rmdir /S /Q %SOURCEDIR%\examples\
+    del /Q %SOURCEDIR%\snippets\admonitions.md
 ) else (
-	@echo Generating API index...
```
Comment (niksirbi) [inline docs/make.bat:37]: @ not really needed in front of `echo`, because of the `@ECHO OFF` at the top of `make.bat`.

**Comment by sfmig** [comment]
I like it! You took it to the next level! 🚀 

> we now have to maintain an entire script

True! But maybe maintaining a script is better than remembering both sources of text need to be in sync?

> we may build upon it to do some more intelligent parsing from README.md to index.md, in a way that further reduces duplication.

That sounds like a nice idea.

**Comment by lochhh** [comment]
> ## [Quality Gate Passed](https://sonarcloud.io/dashboard?id=neuroinformatics-unit_movement&pullRequest=335) **Quality Gate passed**
> Issues  [1 New issue](https://sonarcloud.io/project/issues?id=neuroinformatics-unit_movement&pullRequest=335&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true)

Just a reminder here to check this

**Comment by niksirbi** [comment]
Thanks both! I've now refactored the conversion script to reduce its cognitive complexity. I think this is now ready to merge.

**Pull request merged**